packages/circuit-ui/components/CheckboxGroup/index.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright 2022, SumUp Ltd.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { CheckboxGroup } from './CheckboxGroup';
+
+export type { CheckboxGroupProps } from './CheckboxGroup';
+
+export default CheckboxGroup;
```

Comment:
Don't forget to export the `CheckboxGroup` from the `packages/circuit-ui/index.ts` file as well :)

Reply:
Thanks for the pointer! Just did it :)